alternator/streams.cc

```diff
@@ -357,44 +357,35 @@ namespace alternator {
 using namespace std::string_literals;
 
 /**
- * This implementation of CDC stream to dynamo shard mapping
  * is simply a 1:1 mapping, id=shard. Simple, and fairly easy
  * to do ranged queries later.
  * 
  * Downside is that we get a _lot_ of shards. And with actual
  * generations, we'll get hubba-hubba loads of them. 
  * 
- * Improvement ideas: 
- * 1.) Prune expired streams that cannot have records (check ttls)
- * 2.) Group cdc streams into N per shards (by token range)
+ * We would like to group cdc streams into N per shards 
```

Comment:
Adding a comment here because Github doesn't allow me to add comments on the commit message :-(
The word "paging" in the commit message is very ambigous. We do paging of almost everything in Alternator and Scylla, and it's not clear which one is intended here. I didn't understand what "efficient collection skip when doing paging" refers to in the commit message.
Also, please instead of talking about the "sanity" of the parent-child relationship, can you please say a few words about how this parent-child relationship actually works in Scylla? Do we even have this notion?

Reply:
I've updated the comment to include more info.